doc/ChangeLog

```diff
@@ -1,4 +1,159 @@
 ===============================================================
+Tag name: ctsm5.1.dev13?
+Originator(s): @cenlinhe (Cenlin He,UCAR/RAL), slevis (Samuel Levis,UCAR/TSS,303-665-1310)
+Date: Fri Aug 11 14:39:06 MDT 2023
+One-line Summary: SNICAR snow albedo scheme updates
+
+Purpose and description of changes
+----------------------------------
+
+ Notes copied here from the PR #1861:
+
+ A few substantial changes in the SNICAR module for the following updated snow
+ albedo calculation features:
+ - Updated ice optical properties from Flanner et al. (2021), with multiple types
+ for ice refractive indices. The updated SNICAR database files are:
+ /glade/work/cenlinhe/NOAA_CPT_snowAER/CLM_SNICAR_data/snicar_optics_480bnd_c012422.nc
+ /glade/work/cenlinhe/NOAA_CPT_snowAER/CLM_SNICAR_data/snicar_optics_5bnd_c013122.nc
+ - Updated aerosol optical properties from Flanner et al. (2021) with multiple
+ dust types & new BC and OC optics (the updated data are in the new SNICAR
+ database files shown above).
+ - Updated downward solar spectra from Flanner et al. (2021) for multiple
+ condition types.
+ - More accurate radiative transfer solver (adding-doubling) from Dang et al. (2019).
+ - Nonspherical snow grain scheme from He et al. (2017).
+ - BC-snow internal mixing scheme from He et al. (2017).
+ - Dust-snow internal mixing scheme from He et al. (2019).
+ - Hyperspectral (480-band, 10-nm spectral res) capability with all the above features.
+ - New namelist controls for aerosol in snow and additional snow albedo
+ diagnostic output variables.
+
+ Specific notes
+ - Code contributors: Cenlin He (NCAR/RAL) with advice from
+ Dave Lawrence (NCAR/CGD) and Mark Flanner (UMich).
+ - The manuscript to report this update is in preparation (He et al, 2022 JAMES in prep)
```

Comment:
The reference could be updated to this preprint (currently under review in JAMES):
Cenlin He, Mark Flanner, David M Lawrence, et al. New features and enhancements in Community Land Model (CLM5) snow albedo modeling: description, sensitivity, and evaluation. Authorea. June 08, 2023.
DOI:10.22541/essoar.168626390.01530324/v1